src/todomanager/popup.tsx

```diff
@@ -42,36 +46,101 @@ function Popup() {
     );
   };
 
+  const deleteTodo = (id: string) => {
+    setTodos(todos.filter(todo => todo.id !== id));
+  };
+
+  const handleEditTodo = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (editTodo) {
+      setEditTodo({ ...editTodo, content: e.target.value });
+    }
+  };
+  const startEditTodo = (todo: Todo) => {
+    setEditTodo({ id: todo.id, content: todo.content });
+  };
+
+  const saveEditTodo = () => {
+    if (editTodo) {
+      setTodos(
+        todos.map(todo =>
+          todo.id === editTodo.id
+            ? { ...todo, content: editTodo.content }
+            : todo,
+        ),
+      );
+      setEditTodo(null);
+    }
+  };
+
   return (
-    <div>
-      <div className="w-32 h-10">
-        <h1>투두막</h1>
+    <div className="text-[13px] p-2">
+      <div className="w-80 h-10">
+        <h1 className="text-[13px]">투두막</h1>
       </div>
-      <div className="flex-col w-32 h-32">
-        <div className=" h-16 overflow-y-scroll">
+      <div className="flex-col w-full ">
+        <div className="max-w-80 h-[200px] overflow-y-auto">
```

Comment:
```suggestion
        <div className="max-w-80 h-52 overflow-y-auto">
```

Suggest: 200일 필요가 없으면 tailwind 스타일을 사용하면 어떨까요?

[TIP](https://tailwindcss.com/docs/height)

Reply:
넵 수정했습니다! [70bafbc](https://github.com/ToyMakers/todo-mark/pull/11/commits/70bafbc1f6a515ff029abb6ace5c7294746f7203)